### ISSUE TITLE
Unnecessary displayName property on Object with components

### ISSUE THREAD

**Issue by BumbleAd** [hjkcai/swc-plugin-add-display-name#6]
@hjkcai
I found some unexpected functionality that is breaking my app business logic after compilation

For the following code, after compilation `obj` will receive an unexpected `displayName` property.
Is it possible to avoid this addition?

```
const TestComponent = () => {
  return <div>SIMPLE COMPONENT</div>;
};

const obj = {
  PLAY: {
    prop1: "aaaa",
    comp: <TestComponent />,
  },
};
```
[Code Snippet](https://codesandbox.io/p/sandbox/green-silence-6z7yg7?file=%2Fsrc%2FApp.js%3A13%2C3)

**Comment by hjkcai** [comment]
This is indeed a problem of the current algorithm. It looks for top level definitions which contains jsx expressions. That will be marked as a react component, and a displayName property will be added to it.

In your case, obj is a top level variable definition containing an jsx expression. So an unwanted property has been added.

I think I will try to exclude object literals for now. I'm not sure whether there are more cases like this.

**Comment by hjkcai** [comment]
I have published v0.3.1 fixing this issue.
